Content.Client/GameObjects/Components/Movement/ClimbModeComponent.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Content.Shared.GameObjects.Components.Movement;
+using Content.Shared.Physics;
+
+namespace Content.Client.GameObjects.Components.Movement
+{
+    [RegisterComponent]
+    public class ClimbModeComponent : SharedClimbModeComponent
+    {
+        private ICollidableComponent _body = default;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            if (Owner.TryGetComponent(out ICollidableComponent body))
```

Comment:
I'm pretty sure you can point the TryGetComponent out parameter to your `_body` variable.